### PR TITLE
RGRIDT-899: Adding format to date/dateTime columns

### PR DESCRIPTION
**Pull request by glundgren93**
This PR adds format to Date and DateTime columns.

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: code/src/WijmoProvider/Columns/DateColumn.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ namespace WijmoProvider.Column {
 
         public build(): void {
             //Setting date format
-            this.config.format = GridAPI.dateFormat;
+            this.config.format = this.editorConfig.format || GridAPI.dateFormat;
```
Comment (tmbp95) [inline code/src/WijmoProvider/Columns/DateColumn.ts:52]: Is this the only way to get the dateFormat? We should avoid accessing the GridAPI on the wijmoProvider

File: code/src/WijmoProvider/Columns/DateTimeColumn.ts [hunk 2]
```diff
@@ -48,7 +48,9 @@ namespace WijmoProvider.Column {
 
         public build(): void {
             this.editorConfig.timeFormat = 'HH:mm';
-            this.config.format = `${GridAPI.dateFormat} ${this.editorConfig.timeFormat}`;
+            this.config.format =
+                this.editorConfig.format ||
+                `${GridAPI.dateFormat} ${this.editorConfig.timeFormat}`;
```
Comment (tmbp95) [inline code/src/WijmoProvider/Columns/DateTimeColumn.ts:53]: Is this the only way to get the dateFormat? We should avoid accessing the GridAPI on the wijmoProvider

File: code/src/WijmoProvider/Columns/DateColumn.ts (hunk 1 above)
Comment (rugoncalves) [reply code/src/WijmoProvider/Columns/DateColumn.ts:52]: I agree with Tiago. Also these changes are they anyhow typed in OS side? as in, can we have them typed somehow (e.g. enum)?

File: code/src/WijmoProvider/Columns/DateTimeColumn.ts (hunk 2 above)
Comment (rugoncalves) [reply code/src/WijmoProvider/Columns/DateTimeColumn.ts:53]: Agree, we should preferably always use the config, now that we have it. Meaning that when we're loading the configs we should set the value if they come empty, and from them on, the configs are "master".

**Review by rugoncalves — CHANGES_REQUESTED** [review]

**Review by rugoncalves — APPROVED** [review]

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Pull request merged**